CHANGELOG.md

```diff
@@ -2,6 +2,25 @@
 
 ## [Unreleased]
 
+## [2.14.0] - 2018-09-27
+
+### Added
+- Credhub introduced as a user-accessible component (independent of future cf-deployment requirements)
+- Exposed SMTP variable to allow for account creation & password reset
+
+### Changed
+- One Kubernetes service per job now, which will require changes to role manifest YAML 
+- Bumped python & Ruby buildpacks
+- Bumped SLE12 & openSUSE stacks
+
+### Fixed
+- Kubernetes readiness check looks for both apiserver & kubelet now
```

Comment:
We always checked for those, but previous only found them when started by hyperkube. Now we also find them when started by other means.

Reply:
New wording: `Kubernetes readiness check no longer looks for hyperkube explicitly`